packages/components/organisms/f-footer/test/specs/component/desktop/f-footer.component.spec.js

```diff
@@ -1,18 +1,13 @@
-const Footer = require('../../../test-utils/component-objects/f-footer.component');
-const footer = new Footer();
 const forEach = require('mocha-each');
+const Footer = require('../../../../test-utils/component-objects/f-footer.component');
+const footer = new Footer();
 
-describe('f-footer component tests', () => {
+describe('Shared - f-footer component tests', () => {
```

Comment:
This should be `Desktop`. Happy to fix as part of another PR though

Reply:
I'll fix this now as it's a quick fix.